go/vm/geth/geth.go

```diff
@@ -166,12 +166,31 @@ func canTransferFunc(stateDB geth.StateDB, callerAddress common.Address, value *
 	return stateDB.GetBalance(callerAddress).Cmp(value) >= 0
 }
 
+type selfDestroyEntry struct {
```

Comment:
The names should be consistent with the other structs and functions. This could also be simplified to just the beneficiary as we discussed offline.